### PR TITLE
Cluster backup: add option to pass service account role ARN when enabling backups 

### LINKED ISSUE #334: Cluster backup: add option to pass service account role ARN when enabling backups
Add serviceAccountRoleARN field to Enable backup in case of Amazon.

### PR DESCRIPTION
**Pull request by sancyx**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | fixes #334 
| License         | Apache 2.0

### What's in this PR?
Regenerate Pipeline API.
Add serviceAccountRoleARN field to Enable backup in case of Amazon.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

**Review by bonifaido — APPROVED** [review]

File: .gen/pipeline/version.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package pipeline
 
-const PipelineVersion = "0.61.0"
+const PipelineVersion = "sa-role-velero"
```
Comment (pregnor) [inline .gen/pipeline/version.go:3]: **Reminder**: don't forget to regenerate the files once a valid Pipeline version tag is available.

File: Makefile [hunk 2]
```diff
@@ -21,7 +21,7 @@ endif
 TEST_FORMAT = short-verbose
 endif
 
-PIPELINE_VERSION = 0.61.0
+PIPELINE_VERSION = sa-role-velero
```
Comment (pregnor) [inline Makefile:24]: **Reminder**: don't forget to update the Pipeline version to a valid tag once available.

File: internal/cli/command/cluster/integratedservice/services/backup/enable.go [hunk 3]
```diff
@@ -223,19 +223,35 @@ func buildEnableRequestInteractively(banzaiCli cli.Cli, options enableOptions) (
...
 
+	serviceAccountRoleARN := ""
 	useClusterSecret := false
 	if options.ClusterCloud() == amazonType {
 		err = input.DoQuestions([]input.QuestionMaker{
-			input.QuestionConfirm{
+			input.QuestionInput{
+				QuestionBase: input.QuestionBase{
+					Message: "Service Account Role ARN to use for Velero",
+					Help:    "This IAM role ARN will be used to annotate service account for Velero and should be associated with relevant snapshot permissions."},
```
Comment (pregnor) [inline internal/cli/command/cluster/integratedservice/services/backup/enable.go:233]: **Recommended**: we **SHOULD** highlight the optional nature of the service account role ARN for the users to convey the possible usages more clearly.

```suggestion
					Help:    "This optional IAM role ARN will be used to annotate the service account for Velero and should be associated with the relevant snapshot permissions. Alternatively, when no service account role ARN is specified, either an optional cluster secret can be deployed or a node instance role can be extended with the snapshot permissions for creating volume snapshots with Velero."},
```

**Review by pregnor — APPROVED** [review]
Please rebase onto latest master because in the meantime, the security group changes were merged with the 0.63.0 Pipeline update.

**Pull request merged**